packages/augur-ui/src/modules/market-cards/common.tsx

```diff
@@ -29,17 +29,22 @@ export interface OutcomeProps {
   lastPricePercent?: number;
   invalid?: Boolean;
   index: number;
+  min: number;
+  max: number;
 }
 
-export const Outcome = (props: OutcomeProps) => (
-  <div className={classNames(Styles.Outcome, {[Styles.invalid]: props.invalid, [Styles[`Outcome-${props.index}`]]: !props.invalid})}>
-  	<div>
-    	<span>{props.description}</span>
-    	<span>{props.lastPricePercent.formatted}%</span>
+export const Outcome = (props: OutcomeProps) => {
+  const percent = calculatePosition(props.min, props.max, props.lastPricePercent.value);
```

Comment:
If maxPriceBigNumber and minPriceBigNumber is used then the component already has big numbers to work with

Reply:
looks good